sound/soc/soc-topology.c

```diff
@@ -578,7 +578,9 @@ static int soc_tplg_kcontrol_bind_io(struct snd_soc_tplg_ctl_hdr *hdr,
 
 	if (le32_to_cpu(hdr->ops.info) == SND_SOC_TPLG_CTL_BYTES
 		&& k->iface & SNDRV_CTL_ELEM_IFACE_MIXER
-		&& k->access & SNDRV_CTL_ELEM_ACCESS_TLV_READWRITE
+		&& (k->access & SNDRV_CTL_ELEM_ACCESS_TLV_READWRITE
+		|| k->access & SNDRV_CTL_ELEM_ACCESS_TLV_WRITE
+		|| k->access & SNDRV_CTL_ELEM_ACCESS_TLV_READ)
```

Comment:
``k->access & SNDRV_CTL_ELEM_ACCESS_TLV_READWRITE`` should be enough since ``#define SNDRV_CTL_ELEM_ACCESS_TLV_READWRITE     (SNDRV_CTL_ELEM_ACCESS_TLV_READ|SNDRV_CTL_ELEM_ACCESS_TLV_WRITE)``